Sources/PostgresNIO/New/PostgresQuery.swift

```diff
@@ -43,6 +43,7 @@ extension PostgresQuery {
             self.sql.append(contentsOf: literal)
         }
 
+        @_disfavoredOverload
```

Comment:
I know this is not what you'd like to see, but i couldn't come up with a better solution.

Without this, as far as I could figure out, the new collection interpolate function will need a parameter name which will make it so users need to interpolate their values like `"\(parameterName: myValues)"`. 
The problem with this is that users can still do `"\(myValues)"` and it will compile, but it will fail when making the requests to Postgres.
With this new way though, the compiler will be able to figure that out and automatically interpolate the value as a sequence.

Reply:
Outdated, see https://github.com/vapor/postgres-nio/pull/410#issuecomment-1732296628